.github/workflows/node.js.yml

```
@@ -0,0 +1,30 @@
+# This workflow will do a clean install of node dependencies, cache/restore them, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Node.js CI
```

```suggestion
name: Semiotic
```

We may potentially have a separate workflow for Docs/E2E so we better distinguish them from the start